### ISSUE TITLE
should locator map allow referencing CoRIM by `corim-map.id`?

### ISSUE THREAD

**Issue by thomas-fossati** [ietf-rats-wg/draft-ietf-rats-corim#73]
At present locator allows reference by URLs.  Should we allow reference by `corim-map.id` implying that the links are local to the verifier?

**Comment by nedmsmith** [comment]
wouldn't the uri include the corim.id? 
e.g., https://my.domain.com/corims/<corim.id>

**Comment by thomas-fossati** [comment]
It could, but that's a deployment / implementation choice.  What I'm looking for is a way to *directly* use CoRIM IDs, without making them relative URLs (to which base URL?) or defining a URL convention (which seems to go counter [BCP190](https://www.rfc-editor.org/rfc/rfc7320.html)).

**Comment by nedmsmith** [comment]
> It could, but that's a deployment / implementation choice. What I'm looking for is a way to _directly_ use CoRIM IDs, without making them relative URLs (to which base URL?) or defining a URL convention (which seems to go counter [BCP190](https://www.rfc-editor.org/rfc/rfc7320.html)).

What is the use case for direct references? Is there a structure that contains multiple CoRIMs?

**Comment by thomas-fossati** [comment]
> Is there a structure that contains multiple CoRIMs?

Yes, something like a batch submission using a `multipart/mixed`

**Comment by nedmsmith** [comment]
Seems reasonable.

**Comment by deeglaze** [comment]
Poking at this to see what folks are thinking to move this forward. Given that we want to provide reference values for our firmware as built for multiple technologies, it'll need to be distributed with a RIM bundle. The TCG information model for RIMs allows for the dependency structure through dependent-rims

```
corim-bundle = [ + corim ]
```
And recommend using `urn:uuid:`corim-id to reference locally-bundled CoRIMs?
This would need yet another content type.
